.travis.yml

```diff
@@ -10,7 +10,7 @@ services:
 before_script:
   - cat test/fixtures/data.sql | mysql -u travis
   - npm install -g gulp
-  - npm install -g lux-framework
+  - npm link
```

Comment:
This is definitely a must. Thanks man! Can you do this in a separate PR? It would be nice to have record of this change after we squash and merge this PR.